debian/control

```diff
@@ -0,0 +1,44 @@
+Source: cobbler
+Section: admin
+Priority: optional
+Maintainer: The Cobbler Authors <cobbler.project@gmail.com>
+Build-Depends:
+ git-core,
+ python3,
+ python3-distro,
+ python3-setuptools,
+ python3-netaddr,
+ python3-requests,
+ python3-schema,
+ python3-cheetah,
+ python3-dns,
+ python3-sphinx,
+ dh-python,
+ debhelper
+Standards-Version: 4.6.1
+Version: 3.4.0
+Homepage: https://cobbler.github.io/
+
+Package: cobbler
+Architecture: all
+Depends:
+ systemd,
+ apache2 | httpd,
```

Comment:
If I'm not mistaken, there is no httpd package on debian like systems, only apache2.

Reply:
Thanks for having a look!

Reply:
This may be true, but the problem is, that we are missing configurations for other webservers like nginx etc. We only have apache configs in the config directory. So either we say it's just compatible with apache or we add config examples for all webservers.

Reply:
So I would love to have examples for all webservers but I have long-term plans to cut out `mod_wsgi` (and thus apache) anyway. So I'd rather ignore the linter in this case as we have a solid reason for that. Additionally I read that `mod_wsgi` and Nginx don't work well in all scenarios, however I havn't looked to deep if this was already fixed (I am referring to http://blog.dscpl.com.au/2009/05/blocking-requests-and-nginx-version-of.html).

Reply:
http://nginx.org/en/docs/http/ngx_http_uwsgi_module.html suggests it should be working nowadays.

Reply:
Okay then if that is fixed I believe we should have it pretty simple running Cobbler behind Nginx and thus adding a config should be not that hard. Any other webserver to keep in mind?

Reply:
Theoretically, all those listed on https://packages.debian.org/de/sid/httpd. But with apache and nginx we have the most of them included.

Reply:
The best bet to support this in a "joint" fashion is switching to uWSGI imho. Then the config from both NGINX and Apache2 will be similar and we don't split ourselves anymore. The workers we need to set to 1 since we are in reality a WSGI application. Right?

See: https://uwsgi-docs.readthedocs.io/en/latest/Apache.html#mod-proxy-uwsgi

Reply:
Here are two good tutorials on how to actually do this:

- https://www.digitalocean.com/community/tutorials/how-to-set-up-uwsgi-and-nginx-to-serve-python-apps-on-ubuntu-14-04
- https://kevalnagda.github.io/flask-app-with-wsgi-and-nginx